### ISSUE TITLE
scylla_prepare: 'get_set_nic_and_disks_config_value' is not defined

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla#7276]
*Installation details*
Scylla version (or git commit hash):  scylla.x86_64 0:666.development-0.20200922.c1e8d077a  
Cluster size: 4
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS7

### Description

https://github.com/scylladb/scylla/pull/7102/files (Simplify scylla_util.py #7102) moved `get_set_nic_and_disks_config_value()` from `scylla_utils.py` to `dist/common/scripts/scylla_sysconfig_setup`.

But it still uses by scylla_prepare.

### Test info
- test id: 0e6fb270-5742-45af-8f72-96501de0b0c5
- db log: https://cloudius-jenkins-test.s3.amazonaws.com/0e6fb270-5742-45af-8f72-96501de0b0c5/20200923_004854/db-cluster-0e6fb270.zip

### Scylla Log

```
2020-09-23T00:29:53+00:00  rolling-upgrade-upg-mast-centos-db-node-0e6fb270-0-3
  !INFO    | scylla_prepare: Exception occurred while creating perftune.yaml: name 'get_set_nic_and_disks_config_value' is not defined
2020-09-23T00:29:53+00:00  rolling-upgrade-upg-mast-centos-db-node-0e6fb270-0-3
  !INFO    | scylla_prepare: To fix the error, please re-run scylla_setup.
2020-09-23T00:29:53+00:00  rolling-upgrade-upg-mast-centos-db-node-0e6fb270-0-3
  !NOTICE  | systemd: scylla-server.service: control process exited, code=exited status=1
```

/CC @roydahan @syuu1228

**Comment by syuu1228** [comment]
This should fixed by https://github.com/scylladb/scylla/commit/eae2aa58faa7e8f04e000887766b9b3c47038794

**Comment by roydahan** [comment]
Looks like it's fixed when using latest master.

**Comment by slivne** [comment]
@penberg can you please check this is only in  master (4.3)

**Comment by penberg** [comment]
@roydahan You asked whether this issue needs backport to 4.2.

The commit eae2aa58faa7e8f04e000887766b9b3c47038794 ("dist/common/scripts: move back get_set_nic_and_disks_config_value to scylla_util.py") fixes a bug in commit 79d8192dc7d160408cc96a9faf350df425913118 ("dist/common/scripts: move functions that are only called from single file"), which is _not_ in `branch-4.2`.

So no need for a backport, the issue does not happen in 4.2, and I am removing the "backport candidate" label.
